src/src/tx_main.cpp

```diff
@@ -541,19 +543,35 @@ void ICACHE_RAM_ATTR SendRCdataToRF()
       otaPkt.std.type = PACKET_TYPE_MSPDATA;
       if (OtaIsFullRes)
       {
-        otaPkt.full.msp_ul.packageIndex = MspSender.GetCurrentPayload(
```

Comment:
From here down looks to revert some changes in https://github.com/ExpressLRS/ExpressLRS/pull/2798